### PR TITLE
style: use new code style

### PR DESCRIPTION
**Pull request by martiliones**
Change log:
- The project uses google eslint config with some disabled rules
- The code will be placed in the `src` directory
- Linter runs on commit
- Commit messages should follow [the rules](https://github.com/conventional-changelog/commitlint/#what-is-commitlint)

> The lint command works as a stub and only checks its config for the duration of the review.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+- Make sure tests pass!
+
+- Commit messages must follow the [commit message convention](./commit-convention.md). Commit messages are automatically validated before commit (by invoking [Git Hooks](https://git-scm.com/docs/githooks) via [husky](https://github.com/typicode/husky)).
+
+- No need to worry about code style as long as you have installed the dev dependencies - modified files are automatically formatted with Prettier on commit (by invoking [Git Hooks](https://git-scm.com/docs/githooks) via [husky](https://github.com/typicode/husky))..
+
+## Development Setup
+
+You will need [Node.js](https://nodejs.org) **version 16+**.
```
Comment (adamant-al) [inline CONTRIBUTING.md:27]: Can we use Nodejs 14+?

Comment (martiliones) [reply CONTRIBUTING.md:27]: @adamant-al 16 is the current LTS version of NodeJS. We can easily use 14+ to support older projects

File: package.json [hunk 2]
```diff
@@ -4,22 +4,25 @@
 	"description": "REST API for ADAMANT Blockchain",
 	"main": "index.js",
 	"scripts": {
-		"test": "echo \"Error: no test specified\" && exit 1"
+		"test": "jest",
+		"lint": "eslint .eslintrc.js",
+		"lint:fix": "eslint --fix .eslintrc.js",
+		"prepare": "husky install"
 	},
 	"author": "RomanS, Aleksei Lebedev <devs@adamant.im> (https://adamant.im)",
```
Comment (adamant-al) [inline package.json:12]:
```suggestion
	"author": "ADAMANT Foundation <devs@adamant.im> (https://adamant.im)",
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (adamant-al) [reply CONTRIBUTING.md:27]: v14 EOL is 2023-04-30, I think we should support it.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+- If adding a new feature add accompanying test case.
+
+- It's OK to have multiple small commits as you work on the PR - GitHub can automatically squash them before merging.
+
+- Make sure tests pass!
+
+- Commit messages must follow the [commit message convention](./commit-convention.md). Commit messages are automatically validated before commit (by invoking [Git Hooks](https://git-scm.com/docs/githooks) via [husky](https://github.com/typicode/husky)).
+
+- No need to worry about code style as long as you have installed the dev dependencies - modified files are automatically formatted with Prettier on commit (by invoking [Git Hooks](https://git-scm.com/docs/githooks) via [husky](https://github.com/typicode/husky))..
```
Comment (adamant-al) [inline CONTRIBUTING.md:23]:
```suggestion
- No need to worry about code style as long as you have installed the dev dependencies - modified files are automatically formatted with Prettier on commit (by invoking [Git Hooks](https://git-scm.com/docs/githooks) via [husky](https://github.com/typicode/husky)).
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (martiliones) [reply CONTRIBUTING.md:27]: Oh, this version is for _development_. I think it's better to leave it like this

Comment (adamant-al) [reply CONTRIBUTING.md:27]: Ok

**Review by gost3228 — APPROVED** [review]

**Comment by MerdyNumber1** [comment]
Perfect

**Review by MerdyNumber1 — APPROVED** [review]

**Review by adamant-al — APPROVED** [review]

**Comment by adamant-al** [comment]
@martiliones Merge it.

**Pull request merged**